ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -136,3 +136,26 @@ def logspace(start, stop, num=50, endpoint=True, base=10.0, dtype=None, axis=0):
         interval = (stop - start) / num
         stop -= interval
     return ivy.logspace(start, stop, num, base=base, axis=axis, dtype=dtype)
+
+
+@handle_jax_dtype
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"0.3.14 and below": ("float16", "bfloat16", )}, 'jax')
+def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None, axis=0):
+    ret = ivy.linspace(start, stop, num, axis=axis, endpoint=endpoint, dtype=dtype)
+    if retstep:
+        if endpoint:
+            num -= 1
+        step = ivy.divide(ivy.subtract(stop, start), num)
+        return ret, step
+    return ret
+    
+
+@handle_jax_dtype
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"0.3.14 and below": ("float16", "bfloat16", )}, 'jax')
+def geomspace(start, stop, num=50, endpoint=True, dtype=None, axis=0):
+    if not endpoint:
+        interval = (stop - start) / num
+        stop -= interval
+    return ivy.geomspace(start, stop, num, axis=axis, dtype=dtype)
```

Comment:
There's no `ivy.geomspace` funciton in our API, could you try and figure out how to express this compositionally with other funcitions?

Reply:
I tried so many combinations to finally fit most of the mathematical concepts related to Geometrical progressions in this program.